<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addweapondialog</class>
 <widget class="QDialog" name="addweapondialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Custom Weapon</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="weaponTypeComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Bows</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ChargeBlades</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DualBlades</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GreatSwords</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gunlances</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hammers</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HeavyBowguns</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>InsectGlaives</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lances</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LightBowguns</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LongSwords</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SwordnShields</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Weapon Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Attack</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="attackSpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="affinityDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Affinity</string>
   </property>
  </widget>
  <widget class="QComboBox" name="elementTypeComboBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Fire</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Water</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ice</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dragon</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Thunder</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>None/Status</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Element Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Element Value</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="elementSpinBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sharpness</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sharpness1ComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Purple</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>White</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Orange</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Red</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="sharpnessButtonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Purple</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>White</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Orange</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Red</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sharpness+1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addweapondialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addweapondialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
